pyrelational/types.py

```diff
@@ -1,7 +1,10 @@
-from typing import Sized
+from abc import ABC
+from typing import Sized, TypeVar
 
 from torch.utils.data import Dataset
 
+T = TypeVar("T")
```

Comment:
Sorry could you explain this to me?

Reply:
So TypeVar is a way of not specifying/enforcing the type